### ISSUE TITLE
a11y: Color Contrast - Global Settings

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13711]
Insufficient color contrast for standard text

## Actual result

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Instances

"Foreground Text Color: #FFFFFF
Background Text Color: #3D98D3
Ratio: 3.16:1 
Main Content
- Show cacerts
- Show rke-metadata-config
- Hide cacerts (available after you activate Show cacerts button) 
- Edit Setting and View in API link text (on hover)(available after you activate More Actions button) 
        
        "

## Expected results

The contrast ratio between foreground text and background for the standard text should be sufficient.

**Comment by rak-phillip** [comment]
Fixed in VPAT. Issue 229
